src/components/markdown-renderer/markdown-extension/sequence-diagram/sequence-diagram-markdown-extension.ts

```diff
@@ -16,4 +19,8 @@ export class SequenceDiagramMarkdownExtension extends CodeBlockMarkdownExtension
   public buildReplacers(): ComponentReplacer[] {
     return [new CodeBlockComponentReplacer(SequenceDiagram, 'sequence')]
   }
+
+  public buildLinter(): Linter[] {
+    return [new SingleLineRegexLinter(/```sequence/, t('editor.linter.sequence'), () => '```mermaid\nsequenceDiagram')]
```

Comment:
How about extracting the magic strings/regex to make this linter similar to the others?

Reply:
In the other cases this was done because the same regex was used by the renderer AND the codemirror extension. This is not the case here. So I don't see why this is necessary here :thinking:

Reply:
Okay, seems a valid reason